README.md

```diff
@@ -200,6 +200,60 @@ jfx {
 }
 ```
 
+Customize Icons
+===============
+
+To customize the icons used in a native bundle, you have to provide the icons for the appropriate bundle.
+The icons must follow the file name convention in order to get picked up.
+
+> Tip: Set the `verbose` setting to true, to log which files are picked up from your deploy directory.
+
+## macOS
+Icon location: `src/main/delploy/macosx`
+
+In macOS you can provide up to three different icons.
+* .app icon
+* volume icon
+* the background of the window, when opening the dmg volume
+
+The icon file name is depended on your `appName` setting of this plugin.
+
+| Type              | Filename                  |
+| :---------------- |:------------------------- |
+| .app icon         | \<appName>.icns           |
+| volume icon       | \<appName>-volume.icns    |
+| volume background | \<appName>-background.png |
+
+The icon sizes should follow the specified sizes.
+http://iconhandbook.co.uk/reference/chart/osx/
+
+## Linux
+Icon location: `src/main/deploy/linux`
+
+For Linux you can provide one icon.
+
+The icon file name is depended on your `appName` setting of this plugin.
+
+| Type              | Filename        |
+| :---------------- |:--------------- |
+| application icon  | \<appName>.png  |
+
+>Whitespaces in the `appName` will be removed in order to lookup for the icon.
+For example a name like 'foo Bar' will lookup for a icon like 'fooBar.png'
+
+## Windows
+Icon location: `src/main/deploy/windows`
```

Comment:
I think this isn't correct. The path should be `src/main/deploy/packages/windows` as can be seen on the [adjusted-launcher-icon example](https://github.com/FibreFoX/javafx-gradle-plugin/tree/2184164223360f927ddb6c0a2c5f8c7fdbce5cca/examples/adjusted-launcher-icon).

Reply:
To be more precise: `src/main/deploy` will be added to the classloader before running some bundler, the `package`-part (without the S at the end) is from the bundlers (its their base subfolder) and the `windows`-part is from the specific bundler itself.

Reply:
You are right about the path. I messed that up when I wrote it. I updated the README file with the correct path.